test/mitmproxy/addons/test_session.py

```diff
@@ -160,6 +160,7 @@ def test_storage_filter(self):
         assert len(s._view) == 4
 
     @pytest.mark.asyncio
+    @pytest.mark.skip
```

Comment:
Is this `skip` a debug left-over?

Reply:
No, this is intentional. We currently don't use the session/db/protobuf code and it's quite a big maintenance burden. The alternative would be removing it from the source entirely, which I'd also be fine with.

Adjusting this is easy when we decide to adopt it, keeping it up-to-date for every small change is burdensome.